lib/AST/ASTPrinter.cpp

```diff
@@ -2157,10 +2159,51 @@ void PrintAST::printAccessors(const AbstractStorageDecl *ASD) {
   indent();
 }
 
+// This provides logic for looking up all members of a namespace. This is
+// intentionally implemented only in the printer and should *only* be used for
+// debugging.
```

Comment:
What do you mean by "should only be used for debugging"? If you ask Xcode for a C++ header's generated interface, it should show how namespace members are imported, shouldn't it?

Reply:
Fair enough. I'll re-word.